code/ATMOSPHERICS/hvac/spaceheater.dm

```diff
@@ -30,6 +54,12 @@
 	update_icon()
 	return
 
+/obj/machinery/space_heater/campfire/stove/New()
+	..()
+	cell.charge = 0
```

Comment:
[spoiler]it does use a cell though, you 'charge it up' by adding wood to the fire. [/spoiler]